### PR TITLE
If retry_on_except is false, raise exception

### PR DESCRIPTION
**Pull request by cdonati**

### CODE REVIEW

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/451/

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/455/

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/461/

File: AppController/lib/app_controller_client.rb [hunk 1]
```diff
@@ -127,6 +127,9 @@ def make_call(time, retry_on_except, callr)
           if retry_on_except
             Kernel.sleep(1)
             retry
+          else
+            raise FailedNodeException.new('Exception encountered while '\
+              "talking to #{@ip}:#{SERVER_PORT}")
```
Comment (nlake44) [inline AppController/lib/app_controller_client.rb]: End the string with a period.

File: AppController/lib/app_manager_client.rb [hunk 2]
```diff
@@ -50,6 +50,9 @@ def make_call(time, retry_on_except, callr)
           if retry_on_except
             Kernel.sleep(1)
             retry
+          else
+            raise FailedNodeException.new('Exception encountered while '\
+              "talking to #{@ip}:#{SERVER_PORT}")
```
Comment (nlake44) [inline AppController/lib/app_manager_client.rb]: Same throughout.

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/517/

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/520/

**Pull request merged**